politeiawww/invoices.go

```diff
@@ -384,6 +385,107 @@ func (p *politeiawww) processInvoiceDetails(invDetails cms.InvoiceDetails,
 	return &reply, nil
 }
 
+// processUserInvoices fetches all invoices that are currently stored in the
+// cmsdb.
+func (p *politeiawww) processUserInvoices(user *user.User) (*cms.UserInvoicesReply, error) {
+
+	dbInvs, err := p.cmsDB.InvoicesByUserID(user.ID.String())
+	if err != nil {
+		return nil, err
+	}
+
+	invRecs := make([]cms.InvoiceRecord, 0, len(dbInvs))
+	for _, inv := range dbInvs {
+		r, err := p.cache.Record(inv.Token)
+		if err != nil {
+			return nil, err
+		}
+
+		invRec, err := convertDatabaseInvoiceToInvoiceRecord(inv)
+		if err != nil {
+			return nil, err
+		}
+		invRec.Username = p.getUsernameById(invRec.UserID)
+
+		// Get raw record information from d cache
+		pr := convertPropFromCache(*r)
+
+		invRec.Files = pr.Files
+		invRec.CensorshipRecord = pr.CensorshipRecord
+		invRec.Signature = pr.Signature
+		invRecs = append(invRecs, *invRec)
+	}
+
+	// Setup reply
+	reply := cms.UserInvoicesReply{
+		Invoices: invRecs,
+	}
+	return &reply, nil
+}
+
+// processAdminInvoices fetches all invoices that are currently stored in the
+// cmsdb.
+func (p *politeiawww) processAdminInvoices(ai cms.AdminInvoices, user *user.User) (*cms.UserInvoicesReply, error) {
```

Comment:
This route will return a valid response if invalid params are used.  Ex. using 13 as the month.

```
$ politeiawwwcli admininvoices --month=13 --year=2019
{
  "invoices": []
}
```

Would prefer for the route to return a user error for invalid params.